src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -13,6 +13,40 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     internal sealed class SourcePropertySymbol : SourcePropertySymbolBase
     {
+        internal bool ContainsFieldKeyword
+        {
+            get
+            {
+                if (CSharpSyntaxNode is not PropertyDeclarationSyntax property)
+                {
+                    return false;
+                }
+
+                if (property.AccessorList is not { } accessorList)
+                {
+                    return false;
+                }
+
+                foreach (var accessor in accessorList.Accessors)
+                {
+                    var body = ((SyntaxNode?)accessor.Body ?? accessor.ExpressionBody);
+                    if (body is null)
+                    {
+                        continue;
+                    }
+
+                    var containsFieldKeyword = body.DescendantTokens()
+                        .Any(t => t.IsKind(SyntaxKind.IdentifierToken) && t.ContextualKind() == SyntaxKind.FieldKeyword && !t.Parent.IsKind(SyntaxKind.AttributeTargetSpecifier));
```

Comment:
>!t.Parent.IsKind(SyntaxKind.AttributeTargetSpecifier) [](http://example.com/codeflow?start=123&length=53)

It is better to check for what we are looking for (i.e. an ```IdentifierNameSyntax```) rather than checking for everything that we are not interested in.